utils/docker/Dockerfile.remote

```diff
@@ -1,25 +1,27 @@
+# To run this file execute:
+# docker build -f Dockerfile.remote . -t Lunarvim:remote
+
 FROM ubuntu:latest
 
-ARG NEOVIM_RELEASE=v0.5.0
+# Build argument to point to correct branch on GitHub
 ARG LVBRANCH=rolling
 
+# Set environment correctly
 ENV DEBIAN_FRONTEND=noninteractive
+ENV PATH="/root/.local/bin:/root/.cargo/bin:/root/.npm-global/bin${PATH}"
 
-# Install apt dependencies
+# Install dependencies and LunarVim
 RUN apt update && \
   apt -y install sudo curl build-essential git fzf python3-dev python3-pip cargo && \
   curl -fsSL https://deb.nodesource.com/setup_16.x | bash - && \
   apt update && \
   apt -y install nodejs && \
-  curl -L -o /tmp/nvim.appimage https://github.com/neovim/neovim/releases/download/${NEOVIM_RELEASE}/nvim.appimage && \
-  chmod u+x /tmp/nvim.appimage && \
-  /tmp/nvim.appimage --appimage-extract && \
-  mv squashfs-root /usr/local/neovim && \
-  ln -s /usr/local/neovim/usr/bin/nvim /usr/bin/nvim && \
   apt clean && rm -rf /var/lib/apt/lists/* /tmp/* && \
-  LVBRANCH=${LVBRANCH} curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/rolling/utils/installer/install.sh | bash -s -- -y
-
-ENV PATH="/root/.local/bin:/root/.cargo/bin:/root/.npm-global/bin${PATH}"
+  curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/${LVBRANCH}/utils/installer/install-neovim-from-release | bash && \
+  LVBRANCH=${LVBRANCH} curl -s https://raw.githubusercontent.com/lunarvim/lunarvim/${LVBRANCH}/utils/installer/install.sh | bash -s -- -y
```

Comment:
Set `--no-install-dependencies` and install them manually if you need them. It will have failed on the rust ones anyway.

Reply:
The container installs the default `rustc` and `cargo` that comes with the `ubuntu:latest`'s package manager. That way it can compile those packages. 

In any case I think ubuntu has a `ripgrep` package (https://packages.ubuntu.com/hirsute/ripgrep) and `fd` (https://packages.ubuntu.com/hirsute/fd-find). Those can be used in the container.

But I could not find ones for `lazygit`. Seems like the authors have stopped maintaining the PPA. 

Maybe the solution is to use an Arch image instead of ubuntu for the container. That way the packages can stay up to date